### ISSUE TITLE
Average over polygon subset

### ISSUE THREAD

**Issue by huard** [bird-house/flyingpigeon#314]
## Description

Instead of only subsetting, take the average over each polygon.

**Comment by nilshempelmann** [comment]
@huard you mean a fieldmean?

**Comment by huard** [comment]
I guess so, the average over all points included in the geometry. I realized that the function does already that, but it shouldn't. It's a subset process, it should return the full grid intersecting with the geometry. I think I mistakenly set aggregate to True, while it should be set to False.

**Comment by nilshempelmann** [comment]
I am not really following you. 
Country and continent subset returns a grid of the selected polygon. If you set mosaik to true than multiple polygons gets retuned as one multypolygon mosaik. 
The spaghetti/uncertainty plot calculates the spatial average. Is that what you want?

**Comment by huard** [comment]
Correct, but I think the wfs_subset did not return a grid on the selected polygon, but an average over each polygon.

What I want here is a single spatial average value for each time step over each polygon. I'm getting Internal Server Error (500) with the master, any idea ?

**Comment by nilshempelmann** [comment]
You want the single spatial average value as a netCDF output? I have planed to wrap up the fieldmean function next week. 
I could add a process for fieldmean calculation.
Never had a server error 500.

**Comment by huard** [comment]
Correct, but I want the mean computed only over a polygon. 

I think we should update pywps to at least 4.2.3. The 4.2.2 version is incompatible with recent owslib.
